target/linux/ath79/image/generic.mk

```
@@ -3414,3 +3440,16 @@ define Device/zyxel_nbg6616
   SUPPORTED_DEVICES += nbg6616
 endef
 TARGET_DEVICES += zyxel_nbg6616
+
+define Device/gainstrong_oolite-v1.0
+	$(Device/tplink-v1)
+	SOC := ar9331
+	DEVICE_VENDOR := GainStrong
+	DEVICE_PACKAGES := kmod-usb-chipidea2
+	DEVICE_MODEL := Oolite V1.0
+	TPLINK_FLASHLAYOUT := 16Mlzma
+	IMAGE_SIZE := 15552k
+	IMAGE/factory.bin := tplink-v1-image factory -s | append-metadata
+	SUPPORTED_DEVICES += oolite-v1.0
```

official device name for 19.07 was `oolite-v1`, before that it was `oolite`
```suggestion
	SUPPORTED_DEVICES += oolite-v1.0 oolite-v1 oolite
```
see 907d67491068890e27f7611da5c5c4d44a672c3c